app/views/follow_requests/create.js.erb

```diff
@@ -0,0 +1,3 @@
+$(".<%= dom_id(@follow_request.sender) %>_<%= dom_id(@follow_request.recipient) %>").replaceWith("<%=j(render 'follow_requests/follow_unfollow', sender: @follow_request.sender, recipient: @follow_request.recipient)%>");
```

Comment:
This is great! You can also create variables for each jquery so that the code is easier to read!